tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Output/Printing/SwiftPrinter.swift

```diff
@@ -87,7 +87,11 @@ public class SwiftPrinter: Printer, SwiftCodePrinter {
         }
     }
 
-    private func printCodingKeys(for properties: [SwiftStruct.Property]) throws {
+    private func printCodingKeys(for properties: [SwiftStruct.Property]) {
+        // TODO: RUMM-1000 should remove this early return clause
+        if properties.isEmpty {
+            return
+        }
```

Comment:
This is not the right place for this patch. `rum-models-generator's` architecture [is build around the stream](https://github.com/DataDog/dd-sdk-ios/pull/386):
```swift
let rumModels = [*.json] → JSONSchema → JSONType → SwiftType -> RUM SwiftType
```
`rumModels` are then passed to `SwiftPrinter()` and `ObjcInteropPrinter()` for code generation. Instead of patching the printer(s), we should **dismiss unsupported types as early as possible in the stream**, so that it is not processed further causing unexpected behaviours. 

Ideally, we should cut it off in `JSONSchemaReader` by explicitly deleting schemas which use unsupported `additionalProperties` modifier. 

As this is temporary and short-living patch, fixing it in `RUMSwiftTypeTransformer` by filtering out the `SwiftStruct` named `"CustomTimings"` might be easier.

In any way, this should not reach the code generation, as it generates potentially wrong API (we may need to support `additionalProperties` in other way than a `struct`).